test/0520-bz1658632-remove-old-sg-indices

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
+
+# test init script tests cleanup of old SG indices
+# https://bugzilla.redhat.com/show_bug.cgi?id=1658632
+LOGGING_NS=${LOGGING_NS:-openshift-logging}
+
+if [ -n "${DEBUG:-}" ] ; then
+    set -x
+fi
+
+source "$(dirname "${BASH_SOURCE[0]}" )/../hack/lib/init.sh"
+source "${OS_O_A_L_DIR}/hack/testing/util.sh"
+os::util::environment::use_sudo
+
+test_name=$(basename $0)
+init_script="0520-bz1658632-remove-old-sg-indices"
+os::test::junit::declare_suite_start ${test_name}
+
+cleanup() {
+    local return_code="$?"
+    set +e
+    if [ $return_code = 0 ] ; then
+        mycmd=os::log::info
+    else
+        mycmd=os::log::error
+    fi
+    $mycmd ${test_name} test finished at $( date )
+    # this will call declare_test_end, suite_end, etc.
+    os::test::junit::reconcile_output
+    exit $return_code
+}
+trap "cleanup" EXIT
+
+routing_allocation(){
```

Comment:
wouldn't we want to `source` the clean up script and use its functions to verify they work?

Reply:
The test should exercise any of the function we create.  I would expect to not need to explicitly source them

Reply:
wouldn't we want to use the `routing_allocation()` from our clean up script rather than have it copied here as well though?